endpoint.go

```diff
@@ -555,33 +445,6 @@ func (ep *endpoint) Delete() error {
 	return nil
 }
 
-func (ep *endpoint) Statistics() (map[string]*sandbox.InterfaceStatistics, error) {
```

Comment:
Actually if #465 is merged, we no longer need Statistics() method in Endpoint. Client can directly access the Endpoint interface and query the stats from it.